### DISCUSSION TITLE
skip_on_failure on Pydantic V2

### DISCUSSION THREAD

**Discussion by MinuraPunchihewa** [pydantic/pydantic#9456] [Question]
Hello,

The `root_validator` that was available with Pydantic V1 allowed a flag called `skip_on_failure`, which allowed other validators to be run even if prior ones failed (when set to `True`).

I was wondering if there was a way to achieve using the `model_validator` offered by V2? I cannot figure it out from the docs.

**Comment by AdeelH** [comment]
I have the same question. Can someone confirm whether `mode='after'` in v2 is equivalent to `skip_on_failure=True` in v1?
